packages/ui/src/testHelpers.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Queries, queries, render, RenderOptions, RenderResult } from '@testing-library/react';
+
+import { ThemeProvider } from './theme';
+
+export const withExpectedError =
+  <Args extends unknown[], Result>(fc: (...args: Args) => Result) =>
+  (...args: Args): Result => {
+    const err = console.error;
+
+    console.error = jest.fn();
+
+    const result = fc(...args);
+
+    console.error = err;
+
+    return result;
+  };
+
+export const renderWithTheme = <
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  Q extends Queries = typeof queries,
+  Container extends Element | DocumentFragment = HTMLElement,
+>(
+  ui: React.ReactElement,
+  options: RenderOptions<Q, Container> = {},
+): RenderResult<Q, Container> => {
+  return render(<ThemeProvider>{ui}</ThemeProvider>, options);
+};
```

Comment:
💭 As `render` is a function with type overload you can preserve it by writing code like the following:
```
const withTheme = <Render extends typeof render>(renderFc: Render) =>
  ((...args: Parameters<Render>) => {
    const [ui, options] = args;

    return renderFc(<ThemeProvider>{ui}</ThemeProvider>, options);
  }) as Render;

export const renderWithTheme = withTheme(render);
```
This way you also don't rely explicitly on current typings of separate render params and return types.